chat_exporter/parse/mention.py

```diff
@@ -43,18 +46,64 @@ class ParseMention:
     def __init__(self, content, guild):
         self.content = content
         self.guild = guild
+        self.markdowns = []
 
     async def flow(self):
+        self.parse_code_block_markdown()
         await self.escape_mentions()
         await self.escape_mentions()
         await self.unescape_mentions()
         await self.channel_mention()
         await self.member_mention()
         await self.role_mention()
         await self.time_mention()
-
+        await self.slash_command_mention()
+        self.reverse_code_block_markdown()
         return self.content
 
+    def parse_code_block_markdown(self):
```

Comment:
Why do we need this here, but also - if we do need it, do we need to dupe it? Can't we access the `ParseMarkdown` class and go straight to the function?